int64[3] sensori
float32 volt
